packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -144,7 +156,7 @@ export const _FileDownloadCards = (props: _FileDownloadCards): JSX.Element => {
                 showSpinner ? (
                   <Spinner size={SpinnerSize.medium} aria-live={'polite'} role={'status'} />
                 ) : (
-                  <IconButton className={iconButtonClassName}>
+                  <IconButton className={iconButtonClassName} ariaLabel={`${file.name} ${localeStrings.downloadFile}`}>
```

Comment:
So if I set my downloadFile string to `Download` this would read `Filename.... Download`, that doesn't sound right semantically.

Reply:
Fixed, now it announces File download button